app/src/main/java/org/mozilla/fenix/components/toolbar/ToolbarComponent.kt

```diff
@@ -7,6 +7,7 @@ package org.mozilla.fenix.components.toolbar
 import android.view.ViewGroup
 import androidx.core.content.ContextCompat
 import kotlinx.android.synthetic.main.component_search.*
+import kotlinx.android.synthetic.main.component_search.view.*
```

Comment:
Is this import being used? One typically uses this form when accessing the view before it's attached.